alcs-api/src/user/user.dto.ts

```diff
@@ -24,11 +20,15 @@ export class CreateOrUpdateUserDto {
 
   @AutoMap()
   @IsString()
-  givenName: string;
+  name?: string;
+
+  @AutoMap()
+  @IsString()
+  givenName?: string;
```

Comment:
why these one optional? this maybe an issue for mentions

Reply:
I see, bceid does not have those.

Reply:
yes, BCeID only has a few fields, we might need to think about how to do initials and other things differently